src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+module Page.Transfer exposing (Model, Msg, init, msgToString, subscriptions, update, view)
+
+import Api
+import Api.Graphql
+import Avatar
+import Bespiral.Scalar exposing (DateTime(..))
+import Eos exposing (Symbol, symbolFromString)
+import Eos.Account as Eos
+import Graphql.Http
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Html.Lazy as Lazy
+import I18Next
+import Icons
+import Page
+import Route
+import Session.Guest as Guest
+import Session.LoggedIn as LoggedIn exposing (External(..))
+import Session.Shared exposing (Shared)
+import Strftime
+import Time
+import Transfer exposing (Transfer, transferQuery)
+import UpdateResult as UR
+import Utils
+
+
+
+-- INIT
+
+
+init : LoggedIn.Model -> Int -> ( Model, Cmd Msg )
+init { shared } transferId =
+    let
+        currentStatus =
+            Loading transferId
+
+        model =
+            { status = currentStatus }
+    in
+    ( model, initCmd shared currentStatus )
+
+
+initCmd : Shared -> Status -> Cmd Msg
+initCmd shared status =
+    case status of
+        Loading transferId ->
+            Api.Graphql.query shared (transferQuery transferId) CompletedTransferLoad
+
+        _ ->
+            Cmd.none
+
+
+
+-- SUBSCRIPTIONS
+
+
+subscriptions : Model -> Sub Msg
+subscriptions _ =
+    Sub.none
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { status : Status
+    }
+
+
+type State
+    = Transferred
+    | Received
+    | NotInvolved -- if the user looking at the transaction is not involved
+
+
+type Status
+    = Loading Int
+    | InvalidId String
+    | LoadFailed (Graphql.Http.Error (Maybe Transfer))
+    | Loaded (Maybe Transfer) State
+
+
+
+-- VIEW
+
+
+view : LoggedIn.Model -> Model -> Html Msg
+view loggedIn model =
+    let
+        t =
+            I18Next.t loggedIn.shared.translations
+    in
+    case model.status of
+        Loading _ ->
+            Page.fullPageLoading
+
+        InvalidId invalidId ->
+            div [ class "container mx-auto px-4" ]
+                [ Lazy.lazy viewHeader loggedIn
+                , div []
+                    [ text (invalidId ++ t "transfer_result.errors.invalid_id") ]
+                ]
+
+        LoadFailed error ->
+            div []
+                [ viewHeader loggedIn
+                , Page.fullPageGraphQLError (t "transfer_result.title") error
+                ]
+
+        Loaded maybeTransfer state ->
+            case maybeTransfer of
+                Just transfer ->
+                    div []
+                        [ Lazy.lazy viewHeader loggedIn
+                        , div [ class "container" ]
```

Comment:
@Pancakem this is not the correct use of a `container`. Usually we use containers to add automatic margins or to contain a certain group of elements in a view. For example this code visualisation on GitHub have lateral margins, that is, is inside a container that allows a max-width.

Since this part have both the green and white backgrounds, we will have to make they always the same width as the screen, so a container at this top level div don't really makes sense

Reply:
Only removing this will solve a lot of the problems you are having related to responsiveness:

<img width="1263" alt="Screen Shot 2019-12-19 at 13 08 48" src="https://user-images.githubusercontent.com/1082127/71190127-c118f880-2262-11ea-8474-5f584fc226e9.png">
<img width="877" alt="Screen Shot 2019-12-19 at 13 08 09" src="https://user-images.githubusercontent.com/1082127/71190129-c118f880-2262-11ea-8f1d-bb19a965862c.png">
<img width="1226" alt="Screen Shot 2019-12-19 at 13 08 23" src="https://user-images.githubusercontent.com/1082127/71190130-c1b18f00-2262-11ea-8493-ed622805a268.png">